scalafix-core/src/main/scala/scalafix/config/FilterMatcher.scala

```diff
@@ -11,6 +11,7 @@ case class FilterMatcher(
     @ExtraName("include") includeFilters: Regex,
     @ExtraName("exclude") excludeFilters: Regex
 ) {
+  def matches(path: AbsolutePath): Boolean = matches(path.toString())
```

Comment:
Should this match absolute paths, or only the portion relative to the sourceroot?